### PR TITLE
[READY] - update tfsec to latest release

### LINKED ISSUE #47: daily builds appear to be broken
daily build workflow breakage:
https://github.com/Nebulaworks/nix-garage/runs/4913402701?check_suite_focus=true#step:4:707

The build appears to be looking for a dependency at `https://github.com/tfsec/tfsec` that has since been moved to `https://github.com/aquasecurity/tfsec`.  (If you visit the first url in your browser, it redirects you to the 2nd url.)

### PR DESCRIPTION
**Pull request by ruebenramirez**
fixes #47 

## Description of PR
- The github.com/tfsec/tfsec repo references no longer resolve.
- The new repo url is github.com/aquasecurity/tfsec.
- These unresolvable go dependencies break the nix build.
- After our downstream CI has been updated to utilize v0.63.1 (latest) tfsec release, we can merge this PR to fix the broken nix-garage daily build.

## Previous Behavior
daily build is broken: https://github.com/Nebulaworks/nix-garage/runs/4913402701?check_suite_focus=true#step:4:707

## New Behavior
Fixes build

### CODE REVIEW

**Review by jshcmpbll — APPROVED** [review]
LGTM :shipit:

**Comment by jshcmpbll** [comment]
@ruebenramirez good to merge?

**Comment by ruebenramirez** [comment]
> @ruebenramirez good to merge?

@jshcmpbll I think we should hold off until chatting with @sarcasticadmin and @davido-nw.  I mostly wanted this ready to go when we make the decision to run with the latest release of tfsec ;)

File: imgs/nwi-pr-utils/default.nix
Comment (sarcasticadmin) [inline imgs/nwi-pr-utils/default.nix:6]: At this point lets just nuke the `custom-tfsec` drv and use the version thats upstream in nixpkgs

I know @davido-nw was going to us to latest relatively soon too

**Review by sarcasticadmin — APPROVED** [review]
@ruebenramirez `approved` based on the idea since I think its time to just start building against newer versions. We only needed the override to get us to start consume this container (which were all good now). Take a look at my comment but it should simplify the build going forward and get us to build whatevers new and shiny up stream.

@jshcmpbll thanks for the review too! This would have been fine to merge either way, just thinking since were going to `latest` we might as well remove anything blocking potential updates in nixpkgs (the pinning override in this case).

Thanks guys for jumping on this, I was surprised to see a PR here this weekend :laughing:

File: imgs/nwi-pr-utils/default.nix
Comment (ruebenramirez) [reply imgs/nwi-pr-utils/default.nix]: Updated to remove the custom-tfsec.

This has been a really great opportunity to get my hands dirty with nix.  I feel like I'm finally getting the hang of how this is all put together finally ;)

**Review by sarcasticadmin — APPROVED** [review]
This looks good! 

Ill merge after you confirm the containers got what it needs and do a fixup on the commits. :shipit:

**Comment by sarcasticadmin** [comment]
:shipit:

**Pull request merged**
